src/client/vid/vid.c

```diff
@@ -489,6 +490,12 @@ VID_CheckChanges(void)
 		// Unblock the client.
 		cls.disable_screen = false;
 	}
+
+	if (vid_refresh->modified)
```

Comment:
Could we use some boolean flag instead cvar `vid_refresh` ? Look like exact value of variable is never used and only checked `modified` status.

Reply:
Can you clarify what you mean with a boolean flag? If you mean something like a global variable, I don't think that would be any more practical. Since the renderer is dynamically loaded with dlopen(), the main program would have to get the address of such a global variable from the renderer lib. The cvar approach, copied from vkQuake2, is simple in the sense that it allows the variable to be shared between the renderer and the server with ease using the existing mechanisms without modifying the interface.

Reply:
I see, thank you.

Reply:
Can't we just use `vid_renderer` for this? We renamed `vid_refresh` to `vid_renderer` when we changed the renderer API to make sure that the renderer selection is empty / at default when starting with an old config.

Reply:
I have tested changing the code to use vid_renderer instead of introducing a new variable and it mostly works. The only unpleasant side-effect of this change is that the client is re-registered just after changing vid_renderer from the console, before they have the chance to use vid_restart. This does not happen when using the video menu, since changing the renderer and hitting "apply" triggers a renderer restart immediately. I'll push the commit so you can check. If you prefer the status quo, we can just drop the new commit.

Reply:
I think we take this way, because it's the least invasive option. Reintroducing `vid_refresh` would confuse users and adding a new function might break the renderer API. We can evaluate the new function at a later time, let's get this into master first. I'll give a last round of tests later today and will merge it after that if nothing comes up.

Reply:
Hey. F.c.ing with ME isn’t AT ALL a smart thing to do;). Not anymore…enough is enough. (Boolean-is he using steroids….). I’m soon 42 and then 31y. with PC. DO NOT UNDERESTIMATE ME…aaaaaaanymore. My time has ARRIVED. Just need my soon new MacBook Pro. Can’t do anything because hacked since 1994….AND NOW: ME TIME…MELTING TIME…FINALLY. Madness NB: The tag should say the most lol.